ios/AudioInputController.m

```diff
@@ -68,8 +83,11 @@ - (OSStatus) prepareWithSampleRate:(double)desiredSampleRate {
 
         NSLog(@"[WebRTCVad] set mode %d", ok);
 
-        // 5 second audio buffer hint
-        [audioSession setPreferredIOBufferDuration:5 error:nil];
+        // 0.064 was arrived at via trial and error. Too large of a buffer and
+        // my bluetooth earbuds would start beeping relentlessly. Too small and
+        // speaker output wouldn't work until I switched to bluetooth and back.
+        // 0.064 seconds yields 3072 samples at 48khz. 2822 at 44.1khz.
+        [audioSession setPreferredIOBufferDuration:0.064 error:nil];
```

Comment:
Probably need some testing around here besides my iPhone 12 Pro + Airpods/Qudelix, but it worked fine for me. Latency seemed fine and the library was determining `isVoice` or not just as well as it was before from what I could tell.

Reply:
hmm wondering here (if possible) if the buffer duration should just be as function of the sample rate. e.g. always enforce 2048 frames, so `IOBufferDuration = 2048 / sampleRate`. Not sure if this would introduce issues with rounding errors or something, or somehow impact the overall results.  And I'm not sure on the context of this if we'd want to use `sampleRate` or `desiredSampleRate`

Reply:
Yep, I did consider that and had it that way up until the very end. The drawback as far as I could see was that we'd be introducing more latency. I doubt anyone's recording at 8khz, but if they were, they'd see 256ms for 2048 samples. The tradeoff of fixed time is more or less samples, and for fixed samples it's more or less latency. Either approach worked.

We would go off of `sampleRate` I think, because we're actually sampling at `sampleRate` always and would need to set a duration based on that.

Reply:
What got me to switch to fixed time was so that there was a predictable latency for all devices no matter what headset you were using, but it would be more efficient to lower latency if we can get the same amount of work done in fewer samples, that's true.

Reply:
That's actually a good point. I was only a little concerned about handling sample counts that don't nicely fit into that buffer duration. This was something I had to deal with game audio capture. It seems like the underlying library is probably handling this gracefully for us so it makes sense to stick with a specific duration.